### PR TITLE
WMP-206 Support: Added remote asset testing

### PR DESCRIPTION
**Pull request by Unthrottled**
I just wanted these things put in.

### CODE REVIEW

File: src/main/java/zd/zero/waifu/motivator/plugin/assets/RemoteAssetManager.kt
Comment (Unthrottled) [inline src/main/java/zd/zero/waifu/motivator/plugin/assets/RemoteAssetManager.kt:72]: Kotlin [data classes](https://kotlinlang.org/docs/reference/data-classes.html#data-classes) have built in hash-code/equals. So we can just use the `Set` API.

Comment (zd-zero) [reply src/main/java/zd/zero/waifu/motivator/plugin/assets/RemoteAssetManager.kt:72]: Right! I just missed streams 😂

**Review by zd-zero — APPROVED** [review]

**Comment by zd-zero** [comment]
Let's merge this once PR #213 gets merged.

**Comment by Unthrottled** [comment]
This is supposed to go into #213

**Comment by Unthrottled** [comment]
Oh, I forgot about the removal of the method. Could you just plug this stuff in to that branch please and close this?

**Pull request merged**
